### PR TITLE
Create join-us/canddiates.md

### PR DESCRIPTION
**Pull request by natalie-hub**
Adding a page for Candidates to review the candidate process

### CODE REVIEW

File: contributors/join-us/candidates.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Thank you for your interest in joining Mattermost! There are a few ways to [join Mattermost](https://handbook.mattermost.com/contributors/join-us#join-us). This page is intended to give candidates an overview of our interview process.
+
+## Typical Interview Process
+
+If your application indicates a strong alignment with the role you've applied for, our Recruitment team will contact you to initiate the interview process as outlined below.
+
+### Candidate Questionnaire
+
+Candidates are sent a questtionate with a set of standard and role-specific questions as part of the candidate application. Candidates that Mattermost's Recruiting or Hiring Manager have contacted directly are asked to complete these questions as part of the first interview schedule. The answers are kept in the candidate's Lever profile.
```
Comment (justinegeffen) [inline contributors/join-us/candidates.md]:
```suggestion
Candidates are sent a questionnaire with a set of standard and role-specific questions as part of the candidate application. Candidates that Mattermost's Recruiting or Hiring Manager have contacted directly are asked to complete these questions as part of the first interview schedule. The answers are kept in the candidate's Lever profile.
```

**Comment by justinegeffen** [comment]
Thanks, @natalie-hub! This is great. I've made edits directly on the file, hope that's ok with you. I mainly changed the tense of the sentences, but not the primary content.

**Comment by natalie-hub** [comment]
@justinegeffen - thank you, this all looks good! 

@teresa-novoa - I think we just need your review before Justine can merge to the handbook. Thank you!

File: contributors/join-us/candidates.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+* The [Contributor Licence Agreement](https://mattermost.org/mattermost-contributor-agreement/) ("CLA") will be required to sign if your take-home project forms part of our [Mattermost Community](https://handbook.mattermost.com/contributors/join-us#open-source-contributors).
+* The [NDA agreement](https://na3.docusign.net/Member/PowerFormSigning.aspx?PowerFormId=75563bac-a0ac-4aef-a434-06d9c33c1f90&env=na3&acct=e5bb8810-00ab-47c2-af80-9d331c19377f&v=2) may be required if your take-home project doesn't require the CLA to be signed. Your Recruiter will confirm this step.
+
+**Gift Card Options**
+
+Mattermost recognizes and appreciates that candidates spend personal time to complete this project. To share our appreciation, candidates recieve a "Thank You" gift card upon completion of the interview project. Candidates can choose from a variety of gift cards, including: Apple iTunes, Google Plan, eBay, Visa Gift Card. If you are located in a region where you aren't able to use these gift card options, please let your Recruiter know. We're happy to work with you to identify a gift card that you can use.
+
```
Comment (teresa-novoa-zz) [inline contributors/join-us/candidates.md:35]: **References stage**
In between final round of interviews, we run references. We use SkillSurvey platform. Candidates should provide 3-4 references, where at least 1 should be a direct Manager.

Comment (teresa-novoa-zz) [reply contributors/join-us/candidates.md:35]: @natalie-hub

**Review by teresa-novoa-zz — COMMENTED** [review]
@natalie-hub I added the references stage, please check, thanks!

File: contributors/join-us/candidates.md (hunk 2 above)
Comment (natalie-hub) [reply contributors/join-us/candidates.md:35]: This is great, thanks @teresa-novoa !

**Review by natalie-hub — COMMENTED** [review]
All looks good!

**Review by teresa-novoa-zz — APPROVED** [review]

**Review by justinegeffen — APPROVED** [review]

**Review by natalie-hub — COMMENTED** [review]
Looks good, thank you!

**Pull request merged**